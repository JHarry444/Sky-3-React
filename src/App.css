.trainer {
  border: 3px, solid, black;
  margin-right: 10px;
  cursor: pointer;
  max-width: 20%;
}