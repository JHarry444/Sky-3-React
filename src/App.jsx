import { useEffect, useState } from 'react';
import './App.css';
import Converter from './components/Converter';
import Counter from './components/Counter';
import Footer from './components/Footer';
import Header from './components/Header';
import LoggedIn from './components/LoggedIn';
import Trainer from './components/Trainer';
import TrainerContainer from './components/TrainerContainer';
import TrainerForm from './components/TrainerForm';
import axios from 'axios';

function App() {

  const [trainers, setTrainers] = useState([]);

  // only run the request on LOAD
  // fancy onload
  useEffect(() => {
    axios.get("http://localhost:8081/trainers")
      .then(res => setTrainers(res.data))
      .catch(err => console.error(err));
  }, []);



  return (
    <div className="App">
      {/* <Header /> */}
      {/* <LoggedIn loggedIn /> */}
      {/* <LoggedIn /> */}
      {/* <Counter /> */}
      <Converter />
      <TrainerForm setTrainers={setTrainers} />
      <TrainerContainer trainers={trainers} />

      <Footer />

    </div>
  );
}

export default App;
