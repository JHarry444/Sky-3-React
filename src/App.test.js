import App from './App';
import { create } from 'react-test-renderer';


test("maffs", () => {
  expect(2).toEqual(1 + 1);
});


test("app matches snapshot", () => {
  const app = create(<App />);

  expect(app.toJSON()).toMatchSnapshot();

});