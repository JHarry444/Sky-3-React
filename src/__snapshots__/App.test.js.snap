// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app matches snapshot 1`] = `
<div
  className="App"
>
  fdjgdfghdgjdfw
  <header>
    <h1>
      This is a header
    </h1>
  </header>
  <p>
    My name is 
    Jordan
  </p>
  <p>
    I am 
    30
     years old
  </p>
  <p>
    I am a 
    Software
     trainer
  </p>
  <p>
    And I live in 
    Staffordshire
  </p>
  <p>
    My name is 
    Piers
  </p>
  <p>
    I am 
    28
     years old
  </p>
  <p>
    I am a 
    Devops
     trainer
  </p>
  <p>
    And I live in 
    Derby
  </p>
  <p>
    My name is 
    243
  </p>
  <p>
    I am 
    26
     years old
  </p>
  <p>
    I am a 
     trainer
  </p>
</div>
`;
